_content/docs/editing.md

```diff
@@ -39,3 +39,33 @@ Getting the site running on your machine locally is easy to do on Mac and Linux.
 A combination of HTML and Markdown (Kramdown variant) is used for marking up our pages. YAML is used for meta data and is documented for shows, committees etc here.
 
 The [Kramdown syntax guide](http://kramdown.gettalong.org/syntax.html) is useful for reference.
+
+## Where Things Live - Basic Editing
+
+When first looking at the repository in a text editor, it can seem quite big, and finding where things live will take a bit of time, yet most of the content storage itself is self explanatory. For a basic contributor the main things you'll need to find are person files, show files and venue files. Individual person files live in the people folder from the main History Project repository. For info on hwo to format a person file, see the [People](https://history.newtheatre.org.uk/docs/person/) page. Files for individual shows live in the shows folder and are sorted by academic year, see the [Shows](https://history.newtheatre.org.uk/docs/show/). Venue files live in the venues folder, see the [Venues](https://history.newtheatre.org.uk/docs/venue/) page.
```

Comment:
"hwo" → "how"

Shows **page**

can we refer to directories like this `_people/` (including the 'code' type formatting)

Reply:
How do you format that correctly?? I either didn't know or have forgotten